### PR TITLE
Accordian inputs

### LINKED ISSUE #173: Add appropriate keyboard interactions for collapse links, buttons
**Description**

Incorporate appropriate keyboard interactions for collapse toggles.  Accordions should follow the WAI ARIA spec for an accordion component (https://www.w3.org/TR/wai-aria-practices-1.1/#accordion), and basic collapse toggle buttons/links should probably follow the spec for "disclosures": https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure 

**Why it's Important**

Currently, collapse toggles cannot be toggled as one would expect to via keyboard.

**Alternatives**

n/a

**Possible Implementation**

Add keyboard interactions as needed, and update the last paragraph of the docs here: https://ucf.github.io/Athena-Framework/components/collapse/#accessibility

**Additional context**

n/a

### PR DESCRIPTION
**Pull request by jmbarne3**
**Description**
Adds input actions when accordion toggles are targeted. Toggling the accordion can be done with `return` or `space`, showing with `up` or `right`, hiding with `down` or `left` and selecting the first or last with `home` and `end` respectively.

**Motivation and Context**
Resolves #173 . See the issue for more details.

**How Has This Been Tested?**
Changes have been tested locally using the example accordion markup in the Athena docs.

**Types of changes**

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [x] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/js/accordian-input.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+function goToLast(target) {
+  const $target = getTarget(target);
+  const $parent = $($target.data('parent'));
+  const $last = $parent.find('[data-toggle=collapse]').last();
+
+  $last.focus();
+}
+
+$('[data-toggle=collapse]').keydown((e) => {
```
Comment (cjg89) [inline src/js/accordian-input.js]: We probably only want to add the non-SPACE/ENTER actions to `[data-toggle=collapse]` elements that are accordion headers specifically.  (https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-8)

File: src/js/accordian-input.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  switch (e.which) {
+    case KEYCODES.SPACE:
+      e.preventDefault();
+      toggleCollapse(e.target);
+      break;
+    case KEYCODES.RIGHT:
+    case KEYCODES.DOWN:
```
Comment (cjg89) [inline src/js/accordian-input.js]: Directional key actions should navigate to a previous or next accordion header according to the keyboard interaction spec, not open/close the collapsible element (https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction)

File: src/js/accordian-input.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  const $parent = $($target.data('parent'));
+  const $first = $parent.find('[data-toggle=collapse]').first();
+
+  $first.focus();
+}
+
+function goToLast(target) {
+  const $target = getTarget(target);
+  const $parent = $($target.data('parent'));
+  const $last = $parent.find('[data-toggle=collapse]').last();
```
Comment (cjg89) [inline src/js/accordian-input.js]: Getting the first/last accordion header could potentially be simplified with something like:
```
const $last = $target.siblings('[data-toggle="collapse"]').last();
```

Comment (jmbarne3) [reply src/js/accordian-input.js]: I'm working on some of the other fixes, but on this one... the collapse toggles aren't necessarily going to be siblings. In the example from the Athena docs I'm using, they're each wrapped in a card, so the `<div class="card">` elements are siblings, but not the toggles inside of them.

**Pull request merged**